### DISCUSSION TITLE
[Section 6.32 - Cognito] Erro Backend

### DISCUSSION THREAD

**Discussion by jacksonpedrini** [chgasparoto/curso-aws-com-terraform#82] [Q&A]
Bom dia,

Estou tendo o problema abaixo na inicialização da backend. Parece que ocorre um erro ao acessar a bucket S3.

```
C:\curso-aws-com-terraform\04-serverless-app>terraform init -backend=true -backend-config="backend.hcl"

Initializing the backend...
╷
│ Error: error configuring S3 Backend: error validating provider credentials: error calling sts:GetCallerIdentity: InvalidClientTokenId: The security token included in the request is invalid.
│       status code: 403, request id: 8785a3ea-e645-4f5b-98cd-3568202bc0c1
```

@chgasparoto pode me dar um help?

**Comment by chgasparoto** [comment]
> Bom dia,
> 
> Estou tendo o problema abaixo na inicialização da backend. Parece que ocorre um erro ao acessar a bucket S3.
> 
> ```
> C:\curso-aws-com-terraform\04-serverless-app>terraform init -backend=true -backend-config="backend.hcl"
> 
> Initializing the backend...
> ╷
> │ Error: error configuring S3 Backend: error validating provider credentials: error calling sts:GetCallerIdentity: InvalidClientTokenId: The security token included in the request is invalid.
> │       status code: 403, request id: 8785a3ea-e645-4f5b-98cd-3568202bc0c1
> ```
> 
> @chgasparoto pode me dar um help?

@jacksonpedrini você precisa verificar se o usuário que o terraform está usando tem as permissões necessárias para acessar e escrever no bucket que vai ser usado como o backend.

**Reply by jacksonpedrini** [reply]
Bom dia @chgasparoto 

Tive que criar um novo usuário e para o terraform, ai deu certo. O usuário atual estava com algum tipo de bug.

**Comment by Douglas1303** [comment]
Eu estou com o mesmo problema. Já criei um novo usuário como o amigo aqui sugeriu mas não funcionou. Alguém achou alguma outra solução??

**Reply by chgasparoto** [reply]
O problema não está relacionado com o usuário em si mas nas permissões que esse usuário tem. Você precisa rever essas permissões e verificar se a ação acima está incluída.
